pkgs/by-name/fo/fork-cleaner/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}: let
+  version = "2.3.1";
+in
+  buildGoModule {
+    name = "fork-cleaner";
+    inherit version;
+
+    src = fetchFromGitHub {
+      owner = "caarlos0";
+      repo = "fork-cleaner";
+      rev = "v${version}";
+      sha256 = "sha256-JNmpcDwmxR+s4regOWz8FAJllBNRfOCmVwkDs7tlChA=";
+    };
+
+    vendorHash = "sha256-QuIaXXkch5PCpX8P755X8j7MeNnbewWo7NB+Vue1/Pk=";
+
+    allowGoReference = true;
+
+    ldflags = [
+      "-s"
+      "-w"
+      "-X main.version=${version}"
+      "-X main.builtBy=nixpkgs"
+    ];
+
+    meta = {
+      description = "Quickly clean up unused forks on your github account";
+      homepage = "https://github.com/caarlos0/fork-cleaner";
+      changelog = "https://github.com/caarlos0/fork-cleaner/releases/tag/${version}";
+      license = lib.licenses.mit;
+      maintainers = with lib.maintainers; [isabelroses];
+      mainProgram = "fork-cleaner";
+    };
+  }
```

This is trivial, but the convention is to use `with lib;` for the `meta` block.
```suggestion
    meta = with lib; {
      description = "Quickly clean up unused forks on your github account";
      homepage = "https://github.com/caarlos0/fork-cleaner";
      changelog = "https://github.com/caarlos0/fork-cleaner/releases/tag/${version}";
      license = licenses.mit;
      maintainers = with maintainers; [isabelroses];
      mainProgram = "fork-cleaner";
    };
  }
```

---

See https://github.com/NixOS/nixpkgs/pull/303997#discussion_r1564643804.

---

Ah, I see. @isabelroses: feel free to ignore this, I'll approve it regardless of what you do with this specific comment.